json/genres.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+      "id": 636,
+      "name": "Ali"
+    },
+    {
+      "id": 145,
+      "name": "Ahmad"
+    }
```

Comment:
All data should be preserved by storing it inside .json files. The application has to allow users to insert new items (in your case MusicAlbums) and categories (in your case Genres), so the stored data can not be statical. On the program's exit, the inserted data should be stored dynamically in a .json file.